File: mario.c
#include <cs50.h>
#include <stdio.h>

int main()
{
    int n;
    do
    {
        n = get_int("Length of pyramid: \n");
    }
    while (n < 1);

    for (int i = 0; i < n; i++)
    {
        // printing spaces before hash
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        // printing n hashes
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        // printing 2 places
        printf("  ");
        // printing n hashes
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int
num = 0
while True:  # ifinite loop
    num = get_int("Height: ")  # input height from user
    if num > 0 and num < 9:  # checks if num between 0 and 9
        for i in range(1, num+1):
            print(" "*(num-i) + ("#"*i) + "  " + ("#"*i))
        break  # loop ends when num between 0 and 8, and half pyramid printing completed
